examples/terraform/aws/output.tf

```diff
@@ -112,7 +112,11 @@ output "kubeone_workers" {
           diskType         = "gp2"
           ## Only applicable if diskType = io1
           diskIops           = 500
-          isSpotInstance     = var.initial_machinedeployment_spotinstances
+          isSpotInstance     = local.initial_machinedeployment_spotinstances
+          ## Only applicable if isSpotInstance is true
+          spotInstanceConfig = {
+            maxPrice = "var.initial_machinedeployment_spotinstances_max_price"
```

Comment:
```suggestion
            maxPrice = var.initial_machinedeployment_spotinstances_max_price
```

Reply:
It expects a string, so this would fail

Reply:
then use `format("%d")`, a string `"var.initial_machinedeployment_spotinstances_max_price"` doesn't represent a price, it's just a string.

Reply:
Updated, PTAL.